eng/config/PublishData.json

```diff
@@ -1,4 +1,63 @@
 {
+  "feeds": {
+      "vssdk": "https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json",
+      "vs-impl": "https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json"
+  },
+  "packages": {
+    "default": {
+      "Microsoft.CodeAnalysis" : "arcade",
```

Comment:
I wanted the script to throw if it found any packages that did not have a properly configured feed, so I kept these mapped to arcade to show that these are intentionally not being uploaded.